src/modules/package-downloader.ts

```diff
@@ -41,10 +40,6 @@ export class PackageDownloader {
     }
 
     async checkoutVersion(): Promise<void> {
-        if (this.packageConfig.version === 'latest') {
```

Comment:
If there any documentation which references this `latest` version usage? If yes we need to update that.